mandatory/3-function-output.js

```diff
@@ -2,13 +2,15 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// This method generate a random umber between 0 and 1 and multiply with 10
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
-
+// This method is for concatenating two world is the same to write word1 + word2
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord.concat(" ", secondWord).concat(" ", thirdWord);
```

Comment:
Take a look at [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) - this is a very common operation in the functional programming style you are leaning towards.